pkg/datapath/linux/probes/probes.go

```diff
@@ -435,6 +436,41 @@ func HaveIPv6Support() error {
 	return nil
 }
 
+// HaveFibLookupSkipNeigh tests whether or not the kernel supports the
+// BPF_FIB_LOOKUP_SKIP_NEIGH flag for bpf_fib_lookup.
+// https://lore.kernel.org/bpf/20230217205515.3583372-1-martin.lau@linux.dev/
+var HaveFibLookupSkipNeigh = sync.OnceValue(func() error {
+	var objs bpfgen.ProbesObjects
+
+	if err := bpfgen.LoadProbesObjects(&objs, &ebpf.CollectionOptions{}); err != nil {
+		return fmt.Errorf("loading collection: %w", err)
+	}
+
+	defer objs.Close()
+
+	ret, err := objs.ProbeFibLookupSkipNeigh.Run(&ebpf.RunOptions{
+		// Newer kernels require that data is at least 14 bytes:
+		// https://github.com/torvalds/linux/commit/6b3d638ca897e099fa99bd6d02189d3176f80a47
+		Data:   make([]byte, 14),
+		Repeat: 1,
+	})
+	var ve *ebpf.VerifierError
+	if errors.As(err, &ve) {
+		if _, err := fmt.Fprintf(os.Stderr, "Verifier error: %s\nVerifier log: %+v\n", err, ve); err != nil {
+			return fmt.Errorf("writing verifier log to stderr: %w", err)
+		}
+	}
+	if err != nil {
+		return fmt.Errorf("loading eBPF collection into the kernel: %w", err)
```

Comment:
This should indicate that the error happened when trying to run the program, as loading has already succeeded previously
```suggestion
		return fmt.Errorf("running probe program: %w", err)
```